editor/plugins/visual_shader_editor_plugin.cpp

```diff
@@ -2129,7 +2129,7 @@ void VisualShaderEditor::_update_nodes() {
 	}
 
 	Array keys = added.keys();
-	keys.sort();
+	keys.sort_custom(callable_mp_static(&StringLikeVariantOrder::compare));
```

Comment:
If we go with the second solution I'd replace the above method with the list version here